six/modules/c++/cphd/unittests/test_pvp_block_round.cpp

```diff
@@ -162,58 +172,79 @@ bool runTest(bool scale, const std::vector<std::complex<T> >& writeData,
 TEST_CASE(testPVPBlockSimple)
 {
     const types::RowCol<size_t> dims(128, 256);
-    const std::vector<std::complex<sys::Int16_T> > writeData =
+    const std::vector<std::complex<sys::Int16_T>> writeData =
```

Comment:
Looks like these are fine in C++11, but do we have to support pre c++11 as well?

Reply:
Not any more.